config/routes.rb

```diff
@@ -340,7 +340,7 @@
     namespace :api do
       post '/verify/v2/password_confirm' => 'verify/password_confirm#create'
       post '/verify/v2/password_reset' => 'verify/password_reset#create'
-      post '/verify/v2/document_capture' => 'verify/document_capture#create'
+      put '/verify/v2/document_capture' => 'verify/document_capture#create'
```

Comment:
We may be able to keep this as `post` and remove the ternary here to always assign `method: 'POST'`:

https://github.com/18F/identity-idp/blob/ec17ed1f7ec2ae0536268e716823d86a2ae3515e/app/javascript/packs/document-capture.jsx#L163

Reply:
> We may be able to keep this as `post` and remove the ternary here to always assign `method: 'POST'`:
> 
> https://github.com/18F/identity-idp/blob/ec17ed1f7ec2ae0536268e716823d86a2ae3515e/app/javascript/packs/document-capture.jsx#L163

Updated in 296b890.